src/app/(pre-login)/logging-in/page.tsx

```diff
@@ -31,7 +31,7 @@ export default async function LoggingIn() {
 	const { egoJwt, authLoading, setAuthLoading, logIn } = useAuthContext();
 	const EGO_TOKEN_URL = `${EGO_API_ROOT}/api/oauth/ego-token?client_id=${EGO_CLIENT_ID}`;
 
-	if (egoJwt) router.push('/landing-page');
```

Comment:
We can also remove the /landing-page component now that this is working

Reply:
https://github.com/icgc-argo/rdpc-ui/pull/108/commits/286e6e8789cff1ac5fea29b0b262aaf2653f4042